sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TestUtils.java

```diff
@@ -804,6 +855,20 @@ static ExtractKeyPhrasesResultCollection getExtractKeyPhrasesResultCollection()
             new TextDocumentBatchStatistics(2, 2, 0, 2));
     }
 
+    static RecognizeLinkedEntitiesResultCollection getRecognizeLinkedEntitiesResultCollection() {
+        // Categorized Entities
+        // TODO: [Service-bugs] after service fixes the null statistics, then use the values and turn on includeStatics.
```

Comment:
what is this todo? the link added is an issue that is closed

Reply:
This todo should be removed and turn on the batch document statistics..